go.mod

```diff
@@ -63,10 +63,12 @@ require (
 	gopkg.in/jcmturner/gokrb5.v5 v5.3.0
 	k8s.io/api v0.24.3
 	k8s.io/apimachinery v0.24.3
+	k8s.io/cli-runtime v0.24.3
```

Comment:
v0.24.10

Reply:
I think it would be better to do all the k8s.io/* bumps on other PRs.
I would keep this PR changes small.